### ISSUE TITLE
Disable-SSL hook does not work with PS7

### ISSUE THREAD

**Issue by warlof** [microsoft/navcontainerhelper#3066]
**Describe the issue**

It appears the following custom hook which normally allow to bypass ssl certificate validation does not work with Powershell 7 environment :

```
        $sslVerificationDisabled = ($protocol -eq "https://")
        if ($sslVerificationDisabled) {
            if (-not ([System.Management.Automation.PSTypeName]"SslVerification").Type)
            {
                Add-Type -TypeDefinition "
                    using System.Net.Security;
                    using System.Security.Cryptography.X509Certificates;
                    public static class SslVerification
                    {
                        private static bool ValidationCallback(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors) { return true; }
                        public static void Disable() { System.Net.ServicePointManager.ServerCertificateValidationCallback = ValidationCallback; }
                        public static void Enable()  { System.Net.ServicePointManager.ServerCertificateValidationCallback = null; }
                    }"
            }
            Write-Host "Disabling SSL Verification"
            [SslVerification]::Disable()
        }
```

For a reason I cannot understand, the HttpClient simply do not care about it.

I detected this while using the following command in order to build an app : `Compile-AppInNavContainer.ps1`
When running it with Powershell 7, I get the following output :

```
Downloading using HttpClient
ERROR Exception calling "GetResult" with "0" argument(s): "The SSL connection could not be established, see inner exception."
Compile-AppInBCContainer Telemetry Correlation Id: c87d90ca-7f37-490a-87f4-e18b247026ed
Exception calling "GetResult" with "0" argument(s): "The SSL connection could not be established, see inner exception."
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\5.0.3\AppHandling\Compile-AppInNavContainer.ps1:488 char:25
+                         throw (GetExtendedErrorMessage $_)
+                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (Exception calling "…e inner exception.":String) [], RuntimeException
+ FullyQualifiedErrorId : Exception calling "GetResult" with "0" argument(s): "The SSL connection could not be established, see inner > exception."
##[error]PowerShell exited with code '1'.
```

However, if it's run with Powershell core or 5.1, everything is working without any issue.

Caution, as I've retrieve the upper hook also in a generic method here https://github.com/microsoft/navcontainerhelper/blob/b495deb26077badc2ae811543967cbe1da8dba03/AppHandling/PsTestFunctions.ps1#L1242

**Comment by freddydk** [comment]
Is in preview now, thanks

**Comment by freddydk** [comment]
Shipped in 5.0.4
